### ISSUE TITLE
Auto insert first selection

### ISSUE THREAD

**Issue by moonstruck** [davidhalter/jedi-vim#391]
I'm using jedi-vim with neovim. So far experience is great. I need help with a jedi behaviour.

When I press . it automatically inserts the first completion. I have tried both 
let g:jedi#popup_select_first = 0
let g:jedi#popup_select_first = 1
option set in nvimrc file.

Is there any workaround to get rid this?

![screenshot from 2015-04-26 02 39 22](https://cloud.githubusercontent.com/assets/127849/7334749/f8787a34-ebbd-11e4-90aa-253e8562ae7a.png)

Another enhancement/behaviour will be great. If selecting any completion automatically inserts it like YouCompleteMe.

Thanks.

**Comment by blueyed** [comment]
This should be fixed by https://github.com/davidhalter/jedi-vim/pull/396.

Can you try the patch and report back?

**Comment by moonstruck** [comment]
On new version it seems this issue has been fixed.
Previously I applied this patch from https://github.com/davidhalter/jedi-vim/pull/392 after getting suggestion from http://www.reddit.com/user/weisenzahn and got it working.

Today I've updated jedi-vim to latest version and not getting this issue any more.

Current jedi-vim version:
commit 401731c98bee450d337812c08de7183234ff4bb1
Merge: b78f1d7 1005d1a
Author: Dave Halter <davidhalter88@gmail.com>
Date:   Tue Apr 28 19:43:14 2015 +0200

    Merge branch 'dev' of github.com:davidhalter/jedi-vim

Thanks for helping. :)
